swagger.json

```diff
@@ -959,6 +959,203 @@
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/fleet/unassigned_driving_segments/assign": {
+            "post": {
+                "tags": [
+                    "Fleet", "Default"
+                ],
+                "summary": "/fleet/unassigned_driving_segments/assign",
+                "description": "Assign an unassigned driving segment to a driver.",
+                "operationId": "assign_unassigned_driving_segment",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                        "name": "assignUnassignedDrivingSegmentParams",
+                        "required": true,
+                        "in": "body",
+                        "schema": {
+                            "type": "object",
+                            "required": [
+                                "vehicle_id",
+                                "start_ms",
+                                "created_at_ms",
+                                "driver_id",
+                            ],
+                            "properties": {
+                                "vehicle_id": {
+                                    "type": "integer",
+                                    "format": "int64",
+                                    "description": "Vehicle ID for the segment.",
+                                    "example": 212014918086169
+                                },
+                                "start_ms": {
+                                    "type": "integer",
+                                    "format": "int64",
+                                    "description": "Milliseconds UNIX time when this segment started.",
+                                    "example": 1523059200000
+                                },
+                                "created_at_ms": {
+                                    "type": "integer",
+                                    "format": "int64",
+                                    "description": "Milliseconds UNIX time when this segment was created.",
+                                    "example": 1523059200000
+                                },
+                                "driver_id": {
+                                    "type": "integer",
+                                    "format": "int64",
+                                    "description": "Driver ID to assign to this segment.",
+                                    "example": 719
+                                }
+                            }
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The unassigned driving segment that was assigned.",
```

Comment:
Is there some complexity here around what "assigned" means? Does the driver need to accept this change before it is final?